### PR TITLE
proposal is in a database now. Use ActiveRecord

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

**Comment by vuntz** [comment]
LGTM

File: crowbar_framework/app/models/ceph_service.rb [hunk 1]
```diff
@@ -271,7 +271,7 @@ def validate_proposal_after_save proposal
     end
 
     unless radosgw_nodes.empty?
-      ProposalObject.find_proposals("swift").each {|p|
+      Proposal.where(barclamp: "swift").each {|p|
```
Comment (MaximilianMeister) [inline crowbar_framework/app/models/ceph_service.rb]: @bkutil does p.status work in this block?

**Comment by jsuchome** [comment]
+1

File: crowbar_framework/app/models/ceph_service.rb (hunk 1 above)
Comment (bkutil) [inline crowbar_framework/app/models/ceph_service.rb]: yes, it should.

**Comment by bmwiedemann** [comment]
:+1: Syntax check OK, makes proposal edit work in my test ; nanuk successfully tested it

**Pull request merged**